Kickstarter-iOS/Views/Controllers/PledgeSummaryViewController.swift

```diff
@@ -145,18 +151,6 @@ private let rootStackViewStyle: StackViewStyle = { (stackView: UIStackView) in
     |> \.spacing .~ Styles.gridHalf(3)
 }
 
-private let termsTextViewStyle: TextViewStyle = { (textView: UITextView) -> UITextView in
```

Comment:
@cdolm92 note the removed code in this file - you will need it when you add this label back when adding the button drawer.